models/public/ssd_resnet50_v1_fpn_coco/ssd_resnet50_v1_fpn_coco.md

```diff
@@ -0,0 +1,72 @@
+# ssd_resnet50_v1_fpn_coco
```

Comment:
It is better to use hypens `-` instead of underscores `_` -  more human-readable

Reply:
it made for consistency with other TF Object Detection API model names in OMZ (e.g. 
https://github.com/opencv/open_model_zoo/tree/master/models/public/faster_rcnn_resnet50_coco
https://github.com/opencv/open_model_zoo/tree/master/models/public/ssd_mobilenet_v1_fpn_coco
https://github.com/opencv/open_model_zoo/tree/master/models/public/ssdlite_mobilenet_v2

Besides that it is original model name in TF

Reply:
I totally understand you, but we need to eliminate `-`-`_` conflict. And about original name - well, we often don't use exact official name.

Reply:
that is a bit tricky, but I think we should tend to use official name and make changes in limited and specific cases only.

Reply:
Normally I'm in favor of dashes, but in this case I think consistency with existing models is more important. Especially since we already have `ssd_mobilenet_v1_fpn_coco`, which is clearly related to this model.